Makefile

```diff
@@ -88,6 +88,14 @@ ifndef GOBIN
 GOBIN=$(shell go env GOPATH)/bin
 endif
 
+# Check Make version (we need at least GNU Make 4.3). Fortunately,
```

Comment:
According to https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners, 
`ubuntu-20.04` is the latest supported version in GH hosted runners, and `ubuntu-20.04` comes with `make 4.2.1` https://ubuntu.pkgs.org/20.04/ubuntu-main-amd64/make_4.2.1-1.2_amd64.deb.html